virttest/qemu_devices/qdevices.py

```diff
@@ -1476,6 +1476,13 @@ def _insert(self, device, addr):
         self.bus[addr] = device
         return []
 
+    def hotplug(self, device):
+        """
+        Hotplug device
```

Comment:
This method won't hotplug anything, so the name and description is misleading. How about to call it `prepare_hotplug` and description `Prepares a device to be hot-plugged into this bus. You need to call "insert" afterwards. Concurrent calls of "prepare_hotplug" and "insert" are not supported.`